code/modules/research/designs/design_types.dm

```diff
@@ -6,1430 +145,893 @@
-	materials = list(MATERIAL_STEEL = 50, MATERIAL_GLASS = 50)
-	build_path = /obj/item/weapon/stock_parts/capacitor/adv
-	sort_string = "CAAAB"
-	price = 500
-
-/datum/design/item/stock_part/super_capacitor
-	name = "super capacitor"
-	id = "super_capacitor"
-	req_tech = list(TECH_POWER = 5, TECH_MATERIAL = 4)
-	materials = list(MATERIAL_STEEL = 50, MATERIAL_GLASS = 50, MATERIAL_GOLD = 20)
-	build_path = /obj/item/weapon/stock_parts/capacitor/super
-	sort_string = "CAAAC"
+/datum/design/item/stock_part/basic_matter_bin
+	name = "matter bin"
+	id = "basic_matter_bin"
+	materials = list(MATERIAL_STEEL = 80)
+	build_path = /obj/item/weapon/stock_parts/matter_bin
+	price = 100
 
 /datum/design/item/stock_part/micro_mani
 	name = "micro-manipulator"
 	id = "micro_mani"
-	req_tech = list(TECH_MATERIAL = 1, TECH_DATA = 1)
 	materials = list(MATERIAL_STEEL = 30)
 	build_path = /obj/item/weapon/stock_parts/manipulator
-	sort_string = "CAABA"
 	price = 100
 
-/datum/design/item/stock_part/nano_mani
-	name = "nano-manipulator"
-	id = "nano_mani"
-	req_tech = list(TECH_MATERIAL = 3, TECH_DATA = 2)
-	materials = list(MATERIAL_STEEL = 30)
-	build_path = /obj/item/weapon/stock_parts/manipulator/nano
-	sort_string = "CAABB"
-	price = 500
-
-/datum/design/item/stock_part/pico_mani
-	name = "pico-manipulator"
-	id = "pico_mani"
-	req_tech = list(TECH_MATERIAL = 5, TECH_DATA = 2)
-	materials = list(MATERIAL_STEEL = 30)
-	build_path = /obj/item/weapon/stock_parts/manipulator/pico
-	sort_string = "CAABC"
+/datum/design/item/stock_part/basic_micro_laser
+	name = "micro-laser"
+	id = "basic_micro_laser"
+	materials = list(MATERIAL_STEEL = 10, MATERIAL_GLASS = 20)
+	build_path = /obj/item/weapon/stock_parts/micro_laser
+	price = 100
 
-/datum/design/item/stock_part/basic_matter_bin
-	name = "matter bin"
-	id = "basic_matter_bin"
-	req_tech = list(TECH_MATERIAL = 1)
-	materials = list(MATERIAL_STEEL = 80)
-	build_path = /obj/item/weapon/stock_parts/matter_bin
-	sort_string = "CAACA"
+/datum/design/item/stock_part/basic_sensor
+	name = "scanning module"
+	id = "basic_sensor"
+	materials = list(MATERIAL_STEEL = 50, MATERIAL_GLASS = 20)
+	build_path = /obj/item/weapon/stock_parts/scanning_module
 	price = 100
 
+/datum/design/item/stock_part/adv_capacitor
+	name = "advanced capacitor"
+	id = "adv_capacitor"
+	materials = list(MATERIAL_STEEL = 50, MATERIAL_GLASS = 50)
+	build_path = /obj/item/weapon/stock_parts/capacitor/adv
+	price = 500
+
 /datum/design/item/stock_part/adv_matter_bin
 	name = "advanced matter bin"
 	id = "adv_matter_bin"
-	req_tech = list(TECH_MATERIAL = 3)
 	materials = list(MATERIAL_STEEL = 80)
 	build_path = /obj/item/weapon/stock_parts/matter_bin/adv
-	sort_string = "CAACB"
 	price = 500
 
-/datum/design/item/stock_part/super_matter_bin
-	name = "super matter bin"
-	id = "super_matter_bin"
-	req_tech = list(TECH_MATERIAL = 5)
-	materials = list(MATERIAL_STEEL = 80)
-	build_path = /obj/item/weapon/stock_parts/matter_bin/super
-	sort_string = "CAACC"
-
```

Comment:
why removing many small parts here?

Reply:
Changed their position in the list. Now it's Tier 1 -> Tier 2 -> Tier 3